code/src/index.css

```diff
@@ -11,3 +11,42 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
```

Comment:
Good job with using styled components for your project and that the CSS here is for more global things :D